gnssanalysis/gn_download.py

```diff
@@ -268,8 +292,13 @@ def check_n_download_url(url, dwndir, filename=False):
         out_f = _Path(dwndir) / filename
         download_url(url, out_f)
 
+    if uncomp:
+        _subprocess.run(["uncompress", f"{out_f}"])
```

Comment:
Yeah, if we are worried about this, I could import a python package instead that can then be used to decompress the file. I guess in this case the `tarfile` package should be fine?

Reply:
If there is a reasonable python package we can use to handle most files I think that would be pretty sensible rather than relaying on the right application being installed on someone's machine. That doesn't need to happen in this PR though.